homework/style.css

```diff
@@ -1,3 +1,131 @@
+* {
+  margin: 0;
```

Comment:
On a mobile device you should not use large left and right margins. Take full advantage of the limited screen width.